Dockerfile

```diff
@@ -1,15 +1,19 @@
 #Dockerfile for a Postfix email relay service
 FROM centos:latest
-MAINTAINER Juan Luis Baptiste juan.baptiste@gmail.com
+LABEL maintainer="Juan Luis Baptiste juan.baptiste@gmail.com"
```

Comment:
Why this change ? whatever the reason it doesn't belong on this PR.

Reply:
It's the proper new way to mark MAINTAINER in docker files.